Prisma saas security remove duplicate incidents (#17904)
demisto/content: Packs/PrismaSaasSecurity/ReleaseNotes/1_2_9.md

#### Integrations
##### SaaS Security (Beta)
- Updated the Docker image to: *demisto/python3:3.10.1.26972*.

> - Fixed an issue with ***fetch-incidents***, which caused incident duplication.

Reviewer: based on older RN
```suggestion
- Fixed an issue where the ***fetch-incidents*** command created duplicate incidents.
```

Author: @yaakovi I wrote it based on `doc-review` templates
![Screen Shot 2022-03-14 at 14 22 06](https://user-images.githubusercontent.com/78307768/158171226-023b0f78-077c-4192-87c5-331380c03bf1.png)
